clients/ui/frontend/src/typeHelpers.ts

```diff
@@ -0,0 +1,179 @@
+import { isEnumMember } from '~/app/utils/utils';
```

Comment:
we have this file present in shared library, we can import it from there, if needed. So we can remove this file as well